public/pages/Snapshots/components/IndexSettingsInput/IndexSettingsInput.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
This component provides an input for overriding or ignoring index setting during a snapshot restore.